main.tf

```diff
@@ -16,10 +16,16 @@ data "aws_ami" "ubuntu" {
   owners = ["099720109477"] # Canonical
 }
 
+module "secrets" {
+  count                 = var.ca_certificate_secret == null ? 0 : 1
+  source                = "./fixtures/secrets"
+  ca_certificate_secret = var.ca_certificate_secret
+}
+
```

Comment:
The secrets fixture should only be used in tests and backing infrastructure, not in the root module.